packages/mobile/src/utils/time.ts

```diff
@@ -258,6 +258,12 @@ export const formatFeedDate = (timestamp: number, i18next: i18nType) => {
   return quickFormat(timestamp, i18next, 'MMM d')
 }
 
+export const formatFeedSectionTitle = (timestamp: number, i18next: i18nType) => {
+  const title = quickFormat(timestamp, i18next, 'MMMM yyyy')
+  const currentYear = new Date().getFullYear().toString()
+  return title.endsWith(currentYear) ? title.substring(0, title.length - 5) : title
```

Comment:
This is ok for now, but is problematic if we localize this format. For some locales the year isn't at the end.

```
> new Date().toLocaleDateString('fr-FR', {
  year: 'numeric',
  month: 'long',
})
< "septembre 2020"
> new Date().toLocaleDateString('ru-RU', {
  year: 'numeric',
  month: 'long',
})
< "сентябрь 2020 г."
```

Could you add a small note?

Reply:
I thought this wouldn't happen because we're explicitly passing `MMMM yyyy` as a format. Updated the code to use a different format if the years match to avoid problems in the future :)